module/apmfasthttp/server.go

```diff
@@ -93,6 +92,10 @@ func (h *apmHandler) handler(ctx *fasthttp.RequestCtx) {
 	}()
 
 	h.requestHandler(ctx)
+
+	if err := setResponseContext(ctx, tx, bc); err != nil {
+		return
+	}
```

Comment:
AFAICS setResponseContext never returns an error, so can you please remove its err result?